spotbugs-tests/src/test/java/edu/umd/cs/findbugs/nullness/JakartaNonnullTest.java

```diff
@@ -0,0 +1,34 @@
+import edu.umd.cs.findbugs.BugCollection;
+import edu.umd.cs.findbugs.test.SpotBugsExtension;
+import edu.umd.cs.findbugs.test.SpotBugsRunner;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
+import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.nio.file.Paths;
+
+import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.emptyIterable;
+
+@ExtendWith(SpotBugsExtension.class)
+class JakartaNonnullTest {
+    @Test
+    void checkedJakartaNonnullReturn_isOk(SpotBugsRunner spotbugs) {
+        BugCollection bugCollection = spotbugs.performAnalysis(
+                Paths.get("../spotbugsTestCases/build/classes/java/main/CheckedJakartaNonnullReturn.class"));
+        assertThat(bugCollection, emptyIterable());
```

Comment:
Can you please assert that, there is no bug of one or several specific bugtypes? So the test won't break if a new bug would be introduced, which finds an unrelated bug in the example.